Upgrade consideration for active_series_custom_trackers_config (#1897)
grafana/mimir: docs/sources/release-notes/v2.1.md

- The store-gateway attributes cache is now enabled by default (achieved by updating the default for `-blocks-storage.bucket-store.chunks-cache.attributes-in-memory-max-items` from `0` to `50000`). This in-memory cache makes it faster to look up object attributes for chunk data. We've been running this optional cache internally for a while and upon a recent configuration audit, realized it made sense to do the same for all users. The increase in store-gateway memory utilization from enabling this cache is negligible and easily justified given the performance gains.

> Configuration for `active_series_custom_trackers_config` has been moved from `ingester` section to `limits` section and the former going to be deprecated from Mimir version 2.3 .

Reviewer: ```suggestion
- Configuration for `active_series_custom_trackers_config` has been moved from `ingester` section to `limits` section and the former going to be deprecated from Mimir version 2.3 .
```

> Configuration for `active_series_custom_trackers_config` has been moved from `ingester` section to `limits` section and the former going to be deprecated from Mimir version 2.3 .

Reviewer 2: ```suggestion
As part of the enhancement to make active series custom trackers configurable on a per-tenant basis, we've moved where they get configured. Users of this feature should migrate from defining `active_series_custom_trackers_config` in the `ingester` section of Grafana Mimir's [YAML configuration]({{< relref "../operators-guide/configuring/reference-configuration-parameters/#configuration-parameters" >}}) to defining it in the `limits` section. Both options are supported for now to give users time to make the change, but configuring `active_series_custom_trackers_config` via the `ingester` section has been marked deprecated and will be removed in Mimir 2.3. Note: Configuring `active_series_custom_trackers_config` in both the `limits` and `ingester` sections will cause Grafana Mimir to fail on startup. 
```

> As part of the enhancement to make active series custom trackers configurable on a per-tenant basis, we've moved where they get configured. Users of this feature should migrate from defining `active_series_custom_trackers_config` in the `ingester` section of Grafana Mimir's [YAML configuration]({{< relref "../operators-guide/configuring/reference-configuration-parameters/#configuration-parameters" >}}) to defining it in the `limits` section. Both options are supported for now to give users time to make the change, but configuring `active_series_custom_trackers_config` via the `ingester` section has been marked deprecated and will be removed in Mimir 2.3. Note: Configuring `active_series_custom_trackers_config` in both the `limits` and `ingester` sections will cause Grafana Mimir to fail on startup. 

Reviewer: I think needs to be a bullet point, given we're in a list.

```suggestion
- As part of the enhancement to make active series custom trackers configurable on a per-tenant basis, we've moved where they get configured. Users of this feature should migrate from defining `active_series_custom_trackers_config` in the `ingester` section of Grafana Mimir's [YAML configuration]({{< relref "../operators-guide/configuring/reference-configuration-parameters/#configuration-parameters" >}}) to defining it in the `limits` section. Both options are supported for now to give users time to make the change, but configuring `active_series_custom_trackers_config` via the `ingester` section has been marked deprecated and will be removed in Mimir 2.3. Note: Configuring `active_series_custom_trackers_config` in both the `limits` and `ingester` sections will cause Grafana Mimir to fail on startup. 
```
